tests/conftest.py

```diff
@@ -185,6 +185,17 @@ def testproject(_project_factory, project, tmp_path):
     return project.load(path, "TestProject")
 
 
+@pytest.fixture
```

Comment:
given that this fixture is only used in `test_brownie_config`, I think it would be better located in that file than in the `conftest.py`.